virttest/libvirt_xml/vm_xml.py

```diff
@@ -1899,6 +1899,24 @@ def remove_numa_cells(self):
             pass  # Element already doesn't exist
         self.xmltreefile.write()
 
+    @staticmethod
+    def from_domcapabilities(domcaps_xml):
```

Comment:
Better method name follow the convention: verb.+ noun format

Reply:
@chunfuwen I prefer this way. I believe the 'fromX' is a common pattern even in the standard library to provide class constructors. I did a [search](https://docs.python.org/3/search.html?q=from&check_keywords=yes&area=default) for confirmation
E.g.
* [datetime.fromX methods](https://docs.python.org/3/library/datetime.html?highlight=from#datetime.date.fromtimestamp)
* [fractions.from_X methods](https://docs.python.org/3/library/fractions.html?highlight=from#fractions.Fraction.from_float)

I chose underscore_style as usual in avocado. The example I found are all `@classmethod` instead of `@staticmethod`. But I prefer to keep `@staticmethod` in order to restrict its usage to non-instances, i.e.
```python
cpu = VMCPUXML.from_domcapabilities(domcapa_xml)
```
Please, let me know your thoughts.